packages/backend/src/execution/execute.test.ts

```
@@ -289,3 +689,129 @@ describe('execute', () => {
+    iterateRecursively(queryWithDeferResult, (x, _) => {
+        if (isDeferredResult(x)) {
+            if (x.status === 'done') {
+                setNestedValue(queryWithDeferResult, _, x.data);
+            }
+        }
+    });
+}
+
+function setNestedValue(obj: any, path: string[], value: unknown) {
+    for (let i = 0; i < path.length - 1; i++) {
+        if (!obj.hasOwnProperty(path[i])) {
```

Biome

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)